Userland/Utilities/du.cpp

```diff
@@ -228,11 +228,11 @@ u64 print_space_usage(ByteString const& path, DuOption const& du_option, size_t
             return 0;
     }
 
-    if (!du_option.apparent_size) {
+    if (du_option.apparent_size || (path_stat.st_blocks == 0 && path_stat.st_size > 0)) {
```

Comment:
What's the significance of the `path_stat.st_size > 0` here? If a file truly is zero-sized, then both branches will yield the same result anyway.

If we end up updating the change to remove that constraint, then this may also be a good time to add a short explaining one-line comment and to update the commit message with `s/Fix displaying/Use apparent/`.

Reply:
all great points! corrected now :)